src/app/helpers/json/deepClone.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Does a deep clone of a JSON object.
+ * Warning: can't clone functions inside the object. Must be 'pure' object.
+ *
+ * @param {Object} originalObj The object to clone.
+ */
+const deepClone = originalObj => JSON.parse(JSON.stringify(originalObj));
```

Comment:
Can we test these? Maybe have a simple check to ensure it does deep clone, and not just a top level clone?